tests/test_result_filter.py

```diff
@@ -46,7 +46,28 @@ def test_filter():
 
     # Queries that match a small part of the request are ignored, postcode and
     # admins matter in relevant matching words.
-    assert not check("101 dalmatiens", names=["101 rue des dalmatiens"], is_address=True)
+    assert not check("101 dalmatiens", names=["101 rue des dalmatiens"], place_type="address")
     assert check(
-        query="Paris 2e", names=["2e Arrondissement"], admins=["Paris"], postcodes=["75002"]
+        query="Paris 2e",
+        names=["2e Arrondissement"],
+        admins=["Paris"],
+        postcodes=["75002"],
+        place_type="admin",
     )
+
+
+def test_rank():
+    rennes = {
+        "names": ["rue de Paris"],
+        "admins": ["Rennes"],
+        "place_type": "street",
+    }
+
+    paris = {
+        "names": ["rue de Rennes"],
+        "admins": ["Paris"],
+        "place_type": "street",
+    }
+
+    assert rank("rue de Paris, Rennes", **rennes) > rank("rue de Rennes, Paris", **rennes)
+    assert rank("rue de Rennes, Paris", **paris) > rank("rue de Paris, Rennes", **paris)
```

Comment:
If i am not mistaken, it would be more interesting to compare rank values for the same query:
```py
    assert rank("rue de Rennes, Paris", **paris) > rank("rue de Rennes, Paris", **rennes)
    assert rank("rue de Paris, Rennes", **rennes) > rank("rue de Paris, Rennes", **paris)
```

Reply:
Nice catch! I'm quite embarrassed as I actually though about it and still made the wrong choice ... :facepalm: